### PR TITLE
made the /menu endpoint to get a json of the menu from the database

### PR DESCRIPTION
**Pull request by gogogo221**
I changed the Menu model and save_menu.py to include a menu_date member in the Menu model so I could access the menus from the database based on the date. 

heres the output for 2022-07-02
image

heres the output for today 2022-03-06
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "python.linting.pylintEnabled": true,
-    "python.linting.enabled": true,
+    "python.linting.enabled": false,
```
Comment (scottmai) [inline .vscode/settings.json]: please lint your code 😭

File: backend/trojan_dining/models.py [hunk 2]
```diff
@@ -42,6 +42,7 @@ class Meta:
 class Menu(models.Model):
         # for queries
         created_at = models.DateField(auto_now_add=True)
+        menu_date = models.DateField(default = None)
```
Comment (scottmai) [inline backend/trojan_dining/models.py]: this is already in the Menu model, we can just call it `date`

File: backend/trojan_dining/tests/test_webscraper.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import os
 import json
-from django.test import TestCase
+import unittest
```
Comment (scottmai) [inline backend/trojan_dining/tests/test_webscraper.py]: Why can't we use django.test?

File: backend/trojan_dining/views.py [hunk 4]
```diff
@@ -32,3 +36,23 @@ def get(self, request):
         sleep(5)
         user = getpass.getuser()
         return JsonResponse({"username": user})
+
+class GetMenu(APIView):
+    def get(self, request):
+        #get the date and if it doesnt exist return ''
+        date = request.GET.get('date', '')
+        #turn it into datetime
+        date = datetime.datetime.strptime(date, "%Y-%m-%d")
```
Comment (scottmai) [inline backend/trojan_dining/views.py]: What happens if the date is malformatted or missing? Can we make sure this doesn't crash the server and return a 400 status code? (`return HttpResponse(status=204)`)

**Review by scottmai — CHANGES_REQUESTED** [review]
Overall looks good, just a couple notes

File: backend/trojan_dining/tests/test_webscraper.py (hunk 3 above)
Comment (gogogo221) [reply backend/trojan_dining/tests/test_webscraper.py]: yea this was me messing up when merging. It's supposed to be django.test

File: backend/trojan_dining/views.py (hunk 4 above)
Comment (gogogo221) [reply backend/trojan_dining/views.py]: done

File: backend/trojan_dining/models.py (hunk 2 above)
Comment (gogogo221) [reply backend/trojan_dining/models.py]: changed it to date

File: .vscode/settings.json (hunk 1 above)
Comment (gogogo221) [reply .vscode/settings.json]: lol sorry. I forgot to do that.

**Review by scottmai — APPROVED** [review]
Awesome!! Ready to merge 😎

**Pull request merged**